src/index.ts

```diff
@@ -78,6 +78,12 @@ export default class TestingFarmAPI {
     );
   }
 
+  async unsafeNewRequest(request: unknown): Promise<unknown> {
+    return newRequestResponseSchema.parse(
```

Comment:
Don't parse the TF response. Just return it. `return await ...`

Reply:
Hopefully, I have fixed it by commit https://github.com/redhat-plumbers-in-action/testing-farm/pull/111/commits/f8b1954b3ccee53977c3fe41c3a69244aa0f5f81